miso-web/src/main/webapp/pages/editRun.jsp

```diff
@@ -184,8 +162,10 @@
     <td>Description:</td>
     <td>
       <c:choose>
-        <c:when test="${not empty run.status and run.status.health.key ne 'Unknown'}"><form:input
-            path="description" disabled="disabled" class="validateable"/></c:when>
+        <c:when test="${run.health.key ne 'Unknown'}">
+           <form:input path="description" disabled="disabled" class="validateable"/>
```

Comment:
This is word of Rob. That if it's unknown, then you need to specify why, otherwise you don't.